frontend/src/Dashboard/CrossingListPage/CrossingList.js

```diff
@@ -39,10 +39,20 @@ class CrossingList extends React.Component {
       return (<div>Error Loading Crossings</div>);
     }
 
```

Comment:
Can we abstract this out into a method which takes a parameter to sort on -- for readability and extensibility?
OR can we use lodash since there are both filtering and sorting needs and that would clean up a good bit of logic and code.

Reply:
I moved the filter logic into shouldHideCrossing. I'm using it to set the hidden property in the map during render. This brings the number of times we iterate through the list down by one.